wg-deployment/charter.md

```diff
@@ -0,0 +1,35 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- kfctl
+- kfdef for each platform
```

Comment:
kfdef is part of kubeflow/manifest. Trying to understand if this WG maintains manifest of some common components. lIke profile-controller? Or we want other WG owners to update that? Currently, there's no WG taking care of those components

Reply:
My thoughts are to form a `wg-control-plane` with an explicit mandate to own and maintain those components. The `wg-deployment` could then merge onto it or be sunset with the `wg-control-plane` taking over. My concern is that if we commit to everything as part of one WG, then we might not be able to deliver as much.

Reply:
@swiftdiaries Agree. I think it's pretty clear now.

Reply:
Is this working group really maintaining `kfdef for each platform`? e.g. AWS, GCP, OpenShift, etc....
Isn't that unbounded scope since the number of platforms could keep growing?